addons/rose/index.js

```diff
@@ -32,7 +32,6 @@ module.exports = {
    */
   contentFor(type, config) {
```

Comment:
thank you all this work!!

small suggestion: can the comment for this hook (line 28-32) be updated to not reference `@hashicorp/ember-flight-icons`?

Reply:
No problem. Thanks!